e2e_tests/tests/cluster/test_users.py

```diff
@@ -140,26 +136,20 @@ def log_out_user(username: Optional[str] = None) -> int:
     child = det_spawn(args)
     child.read()
     child.wait()
-    return cast(int, child.exitstatus)
-
-
-def activate_deactivate_user(
-    active: bool, target_user: str, admin_credentials: Tuple[str, str]
-) -> int:
-    a_username, a_password = admin_credentials
+    child.close()
+    assert child.exitstatus == 0
 
-    child = det_spawn(
-        ["-u", a_username, "user", "activate" if active else "deactivate", target_user]
-    )
-    expected_password_prompt = f"Password for user '{a_username}':"
-    i = child.expect([expected_password_prompt, pexpect.EOF], timeout=EXPECT_TIMEOUT)
-    if i == 0:
-        child.sendline(a_password)
 
-    child.read()
-    child.wait()
-    child.close()
-    return cast(int, child.exitstatus)
+def activate_deactivate_user(active: bool, target_user: str) -> int:
+    command = [
+        "det",
+        "-m",
+        conf.make_master_url(),
+        "user",
+        "activate" if active else "deactivate",
+        target_user,
+    ]
+    return subprocess.call(command)
```

Comment:
I think this can be simplified to get rid of the return code just like we did with `log_in_user` because the result of this function is always checked to be `== 0`.

```suggestion
def activate_deactivate_user(active: bool, target_user: str) -> None:
    command = [
        "det",
        "-m",
        conf.make_master_url(),
        "user",
        "activate" if active else "deactivate",
        target_user,
    ]
    subprocess.run(command, check=True)
```